.gitignore

```diff
@@ -5,3 +5,4 @@
 /.vagrant/
 /venv/
 /tmp/*
+/*.swp
```

Comment:
is this a vim spaw file being ignored, if so, it should be included in the global gitignore on your system - https://help.github.com/articles/ignoring-files/#create-a-global-gitignore